core/src/main/java/com/scalar/db/api/TableMetadata.java

```diff
@@ -182,53 +195,132 @@ private Builder(TableMetadata prototype) {
       secondaryIndexNames.addAll(prototype.secondaryIndexNames);
     }
 
+    /**
```

Comment:
@josh-wong I've also added these Javadocs. Could you please take a look when you have time? 🙇